src/components/PostRegistration.mjs

```diff
@@ -1,3 +1,5 @@
+import Logo from './AppButton.mjs'; 
```

Comment:
No need to import it here since its not being used in this component. I am guessing you had this here for testing purposes. You can remove it

Reply:
removed - completed